### ISSUE TITLE
Django 3 compatibility - ModuleNotFoundError: No module named 'django.utils.lru_cache'

### ISSUE THREAD

**Issue by nicokant** [django-oscar/django-oscar#3422]
### Issue Summary

module `django.utils.lru_cache` is not found, has been removed in Django 3.0 (https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis) 

### Steps to Reproduce

1. Create a new Django project, I used django-cookiecutter to startup.
2. Add django-oscar to dependecies and install dependecies
3. Add oscar settings to config.settings.base.py
4. Add oscar apps (as from the getting started guide)
5. `python manage.py runserver`

Results in:
```
File "/usr/local/lib/python3.8/site-packages/oscar/config.py", line 9, in <module>
from oscar.core.application import OscarConfig
File "/usr/local/lib/python3.8/site-packages/oscar/core/application.py", line 5, in <module>
from oscar.core.loading import feature_hidden
File "/usr/local/lib/python3.8/site-packages/oscar/core/loading.py", line 9, in <module>
from django.utils.lru_cache import lru_cache
ModuleNotFoundError: No module named 'django.utils.lru_cache'
```

### Technical details

* Python version: 3.8
* Django version: 3.0
* Oscar version: 2.0.4

**Comment by nicokant** [comment]
@solarissmoke can i open a PR?

**Comment by solarissmoke** [comment]
Yes please!

**Comment by nicokant** [comment]
seems like the issue has already been solved in 93e7e66c2b30fed1294e336791f8c4f630ae76f1, it's not released yet

**Comment by solarissmoke** [comment]
As yes, indeed it has. It has been released in version 2.1b. Will be releasing 2.1 stable in the next few days.

**Comment by Qoyyuum** [comment]
Waiting for the 2.1 stable release...
